modules/nf-core/ganon/classify/tests/main.nf.test

```diff
@@ -0,0 +1,112 @@
+nextflow_process {
+
+    name "Test Process GANON_CLASSIFY"
+    script "../main.nf"
+    process "GANON_CLASSIFY"
+    config "./nextflow.config"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "ganon"
+    tag "ganon/buildcustom"
+    tag "ganon/classify"
+
+    setup {
+            run("GANON_BUILDCUSTOM") {
+                script "../../buildcustom/main.nf"
+                process {
+                    """
+                    input[0] = [
+                                [ id:'test' ], // meta map
+                                file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true),
+                            ]
+                    input[1] = []
+                    input[2] = []
+                    """
+                }
+            }
+    }
```

Comment:
The problem is two files are generated and are required but aren't in a folder or tarball we can extract in the module itself. This would thus require a untar upstream process anyway which would be just as wasteful. The tests were extremely fast previously (nf-test is much slower than pytest I find...), so it wasn't so bad IMO.

An alternative would be to separate the two files into separate channels, but I would prefer to do this in a separate  PR if you think it is important.

Reply:
I _think_ it's startup time, but not sure yet.